table.csv

```diff
@@ -130,6 +132,7 @@ messagepack,                    serialization,  0x0201,         draft,     Messa
 car,                            serialization,  0x0202,         draft,     Content Addressable aRchive (CAR)
 libp2p-peer-record,             libp2p,         0x0301,         permanent, libp2p peer record type
 libp2p-relay-rsvp,              libp2p,         0x0302,         permanent, libp2p relay reservation voucher
+memorytransport,                libp2p,         0x0309,         permanent, in memory transport for self-dialing and testing; arbitrar
```

Comment:
```suggestion
memorytransport,                libp2p,         0x0309,         permanent, in memory transport for self-dialing and testing; arbitrary
```

See https://github.com/multiformats/multiaddr/blob/master/protocols.csv

Reply:
Oh no! I lost the last character in my copy paste. Thanks for finding him.